simapp/app.go

```diff
@@ -811,3 +814,21 @@ func BlockedAddresses() map[string]bool {
 
 	return modAccAddrs
 }
+
+func makeSendCoinsMsg(addressCodec coreaddress.Codec) func([]byte, []byte, sdk.Coins) (protoiface.MessageV1, error) {
```

Comment:
imho, this makes the wiring of this module quite weird.
can we add docs here on why its needed?

Reply:
Yes I do agree this makes things weird.